distribution/docs/src/main/resources/content/_configuring/logout-theming.adoc

```diff
@@ -0,0 +1,29 @@
+:title: Logout Page Configuration
+:type: configuration
+:status: published
+:parent: Admin Theming Configurations
+:summary: Configure logout pages displayed on ${branding}.
+:order: 01
+
+== {title}
+
+The logout pages is presented to users through the navigation of ${branding} and has a changeable timeout value.
+
+=== Navigating to Logout Configuration
+
+Configure the Login and Logout pages from the ${admin-console}:
+
+. Navigate to the *${admin-console}*.
+. Select *${ddf-security}* Application.
+. Select *Configuration* tab.
+. Select *Logout Page*.
+
+=== Logout Page Customization
+
+The customizable feature of the logout page is the *Logout Page Time Out*. This is the time limit the IDP
+client will wait for a user to click log out on the logout page. Any requests that take longer than this
+time for the user to submit will be rejected.
+
+. *Default value*: 3600000
```

Comment:
We should probably specify what the unit is here as well.

Reply:
fixed